### DISCUSSION TITLE
How to change the ChromiumWebBrowser right-click menu?

### DISCUSSION THREAD

**Discussion by raphm4044** [cefsharp/CefSharp#4107] [General]
Hi, how to change the ChromiumWebBrowser right-click menu?

**Comment by amaitland** [comment]
You need to implement http://cefsharp.github.io/api/100.0.x/html/T_CefSharp_IContextMenuHandler.htm see 
https://github.com/cefsharp/CefSharp/wiki/General-Usage#handlers

There are examples in the project source. 

In future please include if you are using WinForms or WPF.
